### PR TITLE
0.9.10

### PR DESCRIPTION
**Pull request by DaruZero**
# Description

In this release HiveMQ replaces VerneMQ, a simulator for OPC/UA has been added, along with an internet-facing API to connect to factoryinsight, and the possiblity to split custom tags in Grafana.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

- [x] Installing a new instance
- [x] Upgrading an old instance
- [x] Complete data workflow

# Checklist:

- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding changes to the changelog (if needed).
- [x] I have made corresponding notices to the upgrade instructions (if needed)

# Changelog changes

### Features

- HiveMQ instead of VerneMQ
- OPC/UA simulator
- tulip-connector API
- Multiple custom tag splitting in Grafana

### Chores

- Upgraded various dependencies

# Upgrade instructions

[Here](https://www.notion.so/united-manufacturing-hub/Upgrading-v0-9-9-to-v0-9-10-fa49594f05dd45f6b0bb3cd49fea6973)

### CODE REVIEW

**Comment by DaruZero** [comment]
Ready to merge 🚀 
@Scarjit @JeremyTheocharis

**Comment by Scarjit** [comment]
WIP

File: golang/internal/kafka.go [hunk 1]
```diff
@@ -53,7 +53,23 @@ func SetupKafkaTopicProbeConsumer(configMap kafka.ConfigMap) {
...
-		zap.S().Fatalf("Failed to subscribe to topic %s: %s", probeTopicName, err)
+		tempAdmin, errX := kafka.NewAdminClient(&configMap)
+		if errX != nil {
+			zap.S().Errorf("Failed to create KafkaAdminClient: %s", errX)
+			zap.S().Fatalf("Failed to subscribe to topic %s: %s", probeTopicName, err)
+		}
+		_, errX = tempAdmin.CreateTopics(context.Background(), []kafka.TopicSpecification{
+			{
+				Topic:         probeTopicName,
+				NumPartitions: 1,
```
Comment (JeremyTheocharis) [inline golang/internal/kafka.go:64]: istn standard behaviour 3?

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: golang/cmd/mqtt-kafka-bridge/kafka.go [hunk 2]
```diff
@@ -78,40 +102,66 @@ func kafkaToQueue(topic string) {
...
+		}
 
 		payload := msg.Value
-		var json = jsoniter.ConfigCompatibleWithStandardLibrary
 
-		if json.Valid(payload) {
-			kafkaTopic := msg.TopicPartition.Topic
-			validTopic, mqttTopic := internal.KafkaTopicToMqtt(*kafkaTopic)
+		kafkaTopic := *msg.TopicPartition.Topic
+		if json.Valid(payload) || strings.HasPrefix(kafkaTopic, "ia.raw") {
```
[the remark this answers is not in the document]
Comment (Scarjit) [reply golang/cmd/mqtt-kafka-bridge/kafka.go:152]: https://github.com/united-manufacturing-hub/learn.umh.app/pull/203

File: deployment/united-manufacturing-hub/templates/barcodereader/deployment.yaml [hunk 3]
```diff
@@ -56,6 +62,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            - name: MICROSERVICE_NAME
```
[the remark this answers is not in the document]
Comment (Scarjit) [reply deployment/united-manufacturing-hub/templates/barcodereader/deployment.yaml:65]: https://github.com/united-manufacturing-hub/learn.umh.app/pull/204

File: deployment/united-manufacturing-hub/templates/barcodereader/deployment.yaml [hunk 4]
```diff
@@ -68,6 +78,10 @@ spec:
 
             - name: SCAN_ONLY
               value: {{.Values.barcodereader.scanOnly | quote}}
+            {{if .Values._000_commonConfig.debug.enableFGTrace}}
```
[the remark this answers is not in the document]
Comment (Scarjit) [reply deployment/united-manufacturing-hub/templates/barcodereader/deployment.yaml:81]: https://github.com/united-manufacturing-hub/learn.umh.app/pull/204

File: deployment/united-manufacturing-hub/templates/customMicroservices/statefulset.yaml [hunk 5]
```diff
@@ -34,6 +34,17 @@ spec:
             - name: {{$env_value.name}}
               value: {{$env_value.value}}
             {{end}}
+            - name: MY_POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: MICROSERVICE_NAME
```
[the remark this answers is not in the document]
Comment (Scarjit) [reply deployment/united-manufacturing-hub/templates/customMicroservices/statefulset.yaml:41]: https://github.com/united-manufacturing-hub/learn.umh.app/pull/204

File: deployment/united-manufacturing-hub/values.yaml [hunk 6]
```diff
@@ -2262,6 +2014,18 @@ mqttkafkabridge:
...
     repository: unitedmanufacturinghub/kafka-init
     pullPolicy: IfNotPresent
+  resources:
+    requests:
+      cpu: "100m"
+      memory: "500Mi"
+    limits:
+      cpu: "500m"
+      memory: "750Mi"
+  rawMessageLRUSize: 100000
```
[the remark this answers is not in the document]
Comment (Scarjit) [reply deployment/united-manufacturing-hub/values.yaml:1240]: https://github.com/united-manufacturing-hub/learn.umh.app/pull/204

File: deployment/united-manufacturing-hub/templates/kafkatopostgresql/deployment.yaml [hunk 7]
```diff
@@ -73,9 +73,11 @@ spec:
             - name: metrics
               containerPort: 2112
               protocol: TCP
-            - name: pprof
-              containerPort: 1337
+          {{if .Values._000_commonConfig.debug.enableFGTrace}}
+            - containerPort: 1337
+              name: fgtrace
```
[the remark this answers is not in the document]
Comment (Scarjit) [reply deployment/united-manufacturing-hub/templates/kafkatopostgresql/deployment.yaml:78]: Done. "How to use fgtrace"
Currently set to internal

**Review by JeremyTheocharis — APPROVED** [review]

**Review by Scarjit — APPROVED** [review]

**Pull request merged**